klog.go

```diff
@@ -827,7 +827,7 @@ func (l *loggingT) output(s severity, log logr.InfoLogger, buf *buffer, file str
 			os.Stderr.Write(data)
 		}
 
-		if logging.logFile != "" {
+		if l.logFile != "" {
```

Comment:
can we assume that `l.logFile` holds the proper value at this point?
wondering if this change will break someone.